sample-code/README.md

```diff
@@ -1,26 +1,162 @@
-# Sample Code and E2E Test
-
-![e2e-test](https://github.com/SAP/ai-sdk-js/actions/workflows/e2e-tests.yaml/badge.svg)
+# Sample Code
 
 Sample code to demonstrate the usage of the SAP Cloud SDK for AI.
-Also used as basis for running E2E tests.
+Some of the sample code is also used in E2E test.
+
+### Table of Contents
+
+- [Local Deployment](#local-deployment)
+- [Usage](#usage)
+  - [AI API](#ai-api)
+  - [Foundation Models (Azure OpenAI)](#foundation-models-azure-openai)
+  - [Orchestration](#orchestration)
+  - [Langchain](#langchain)
+
+## Local Deployment
+
+1. Run `pnpm install` to install dependencies.
+2. Create a `.env` file in the `sample-code` directory with your AI core service key by adding the following line:
+
+   ```bash
+   AICORE_SERVICE_KEY='SERVICE_KEY_JSON
+   ```
+
+3. Run `pnpm local` to start the server.
+
+## Usage
+
+### AI API
+
+#### Get all Deployments
+
+GET `/ai-api/deployments`
+
+Get all deployments in resource group "default".
+
+#### Create a Deployment
+
+POST `/ai-api/deployment/create`
+
+```json
+{
+  "configurationId": "CONFIGURATION_ID"
+}
+```
+
+Create a deployment with certain configuration ID in resource group "default".
+
+#### Batch Stop Deployments with Given Configuration ID
+
+PATCH `/ai-api/deployment/batch-stop`
+
+```json
+{
+  "configurationId": "CONFIGURATION_ID"
+}
+```
+
+Stop all deployments with a certain configuration ID in resource group "default".
+The request body should be:
+
+#### Batch Delete all Deployments with Status `STOPPED` or `UNKNOWN`
+
+DELETE `/ai-api/deployment/batch-delete`
+
+Delete all deployments with status `STOPPED` or `UNKNOWN` in resource group "default".
+
+#### Get all Scenarios
+
+GET `/ai-api/scenarios`
+
+Get all scenarios in resource group "default".
+
+#### Get all Foundation Models
+
+GET `/ai-api/models`
+
+Get all foundation models in resource group "default".
```

Comment:
[req] I think instead of repeating this `in resource group "default".`, mention at the beginning that unless specified otherwise, default resource group is used